spec/lib/pedicel/validator_spec.rb

```diff
@@ -0,0 +1,513 @@
+require 'pedicel/validator'
+require 'pedicel-pay'
+
+require 'json'
+require 'digest'
+
+valid_ec_public_key = 'MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE0wzW/i0nT3XOeo6srobJRGnUlmqGTFahuHEOw4M9nxUlTBaQUNc8HlN/z1HbepGXTZWDSJB2deCGfhsrOdVryQ=='
```

Comment:
Maybe better placed in a `model_spec_helper.rb`? Just a thought.

Reply:
Well, yeah, but this would be the only thing in there. I like to keep things as local as possible so it's easy to run just one `_spec.rb` file at a time.